packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -87,6 +91,15 @@ const HeaderTools = ({
                 document.querySelector('html').classList.toggle('pf-v5-theme-dark')} />
             </ToolbarItem>
           )}
+          {hasRTLSwitcher && (
+            <ToolbarItem>
+              <Switch id="ws-rtl-switch" label={isRtl ? 'RTL' : 'LTR'} defaultChecked={false} onChange={() =>{
```

Comment:
Sorry, I know I said we could switch the label, but to be consistent with the dark theme switch, and the dark/RTL switch on full page examples, I think it looks better with a static "RTL" switch label.